packages/plugins/foundry-contracts/ChugSplashLocalExecutor.sol

```diff
@@ -0,0 +1,203 @@
+pragma solidity ^0.8.15;
+
+// SPDX-License-Identifier: MIT
+import { ChugSplashBundles, DeploymentState, BundledChugSplashAction, RawChugSplashAction } from "@chugsplash/contracts/contracts/ChugSplashDataTypes.sol";
+import { UD60x18, ud, convert } from "@prb/math/UD60x18.sol";
+
+abstract contract ChugSplashLocalExecutor {
+    /**
+     * @notice Splits up a bundled action into its components
+     */
+    function disassembleActions(actions[] BundledChugSplashAction) public returns (RawChugSplashAction[], uint256[], bytes32[][]) {
```

Comment:
Could you potentially change the `ChugSplashManager` to accept the `BundledChugSplashTarget`/`BundledChugSplashAction` structs in its execution functions? That way we don't need to disassemble the structs. This'll also make unit testing easier in the future

Reply:
We could do that, but then we'd need to do a breaking release or deal with releasing a new manager version. That doesn't seem worth it to me for this, if you want this maybe make an issue for it and we can do it whenever we want to push out a new set of contracts.

Reply:
Sounds good, made a ticke